.eslintrc.js

```diff
@@ -6,21 +6,16 @@ module.exports = {
     project: ['./tsconfig.json'],
   },
   plugins: [
-    '@typescript-eslint',
```

Comment:
Isn't this still required for TS-specific linting?

Reply:
It's included in the `eslint-config-es` config